tests/infrastructure/golden_images/update_boot_source/test_ssp_data_sources.py

```
@@ -42,38 +42,56 @@ def opt_in_status_str(opt_in):
-            wait_timeout=TIMEOUT_10MIN,
-            sleep=5,
-            func=lambda: data_source.source.name != volume_name_before_reconcile,
-        ):
-            if sample:
-                return
-    except TimeoutExpiredError:
-        LOGGER.error(f"dataSource {data_source.name} was not reconciled")
-        raise
+def wait_for_data_source_updated_referenced_volume(
+    data_source: DataSource,
+    check_exact_match: bool,
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)